packages/nocms/test/eslint.spec.js

```diff
@@ -2,5 +2,5 @@ import mochaEslint from 'mocha-eslint';
 
 mochaEslint([
 	'./src/**/*.js',
-	'./test/**/*.js'
+	'./test/*.js'
```

Comment:
Why not in sub-folders? :-D

Reply:
I'm assuming not to lint the test pages code?

Reply:
You may have trouble linting generated/minified js yes

Reply:
Ah ok :)